src/compiler/glue-compiler.cc

```diff
@@ -22,6 +22,12 @@ using namespace spicy::zeek;
 
 namespace builder = hilti::builder;
 
+#if SPICY_VERSION_NUMBER >= 10700
+static inline auto _linker_scope() { return hilti::builder::scope(); }
+#else
+static inline auto _linker_scope() { return hilti::builder::call("hilti::linker_scope", {}); }
+#endif
+
```

Comment:
If #165 you introduced `spicy_compat.h`. It might make sense to add these functions there instead as well.

Reply:
Actually, #164 gets rid of `spicy_compat.h` once more, because of the new separation there. That's why this is all a bit inconsistent across branches, I'll unify that before the JIT removal lands.